cmds/exp/ssh/main.go

```diff
@@ -30,45 +32,77 @@ import (
 	config "github.com/kevinburke/ssh_config"
 	sshconfig "github.com/kevinburke/ssh_config"
 	"golang.org/x/crypto/ssh"
+	"golang.org/x/term"
 )
 
 var (
-	debug      = flag.Bool("d", false, "enable debug prints")
-	keyFile    = flag.String("i", "", "key file")
-	configFile = flag.String("F", defaultConfigFile, "config file")
+	flags = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
This looks a bit like it needs a gofmt pass -- cahn you check?

Reply:
It has been through gofmt already. Because there is a newline between `flags` and `debug`, the declarations will not be lined up by gofmt